assets/style_codebases.scss

```diff
@@ -0,0 +1,348 @@
+---
+---
+
+@import url('https://fonts.googleapis.com/css?family=Muli:300');
+
+:root {
+  --fpc-yellow: #FFF500;
```

Comment:
As a rule we don't abbreviate the name of our organization nor endorse specific fan developed acronyms. These color names can just be that, I suggest find and replacing to simply remove the `fpc-` and use the color names as variables. So `--yellow`.

Reply:
Captured in https://github.com/publiccodenet/publiccode.net/issues/152